src/control/lib/control/auto_test.go

```diff
@@ -22,19 +22,19 @@ import (
 	"github.com/daos-stack/daos/src/control/logging"
 	"github.com/daos-stack/daos/src/control/security"
 	"github.com/daos-stack/daos/src/control/server/config"
-	"github.com/daos-stack/daos/src/control/server/ioserver"
+	"github.com/daos-stack/daos/src/control/server/engine"
 	"github.com/daos-stack/daos/src/control/server/storage"
 )
 
 var (
-	ioCfg = func(t *testing.T, numa int) *ioserver.Config {
-		return ioserver.NewConfig().
+	ioCfg = func(t *testing.T, numa int) *engine.Config {
```

Comment:
Also here.

Reply:
fixed in https://github.com/daos-stack/daos/pull/4611